src/View/StringTemplate.php

```diff
@@ -369,4 +369,46 @@ public function addClass($input, $newClass, $useIndex = 'class')
 
         return $input;
     }
+
+    /**
+     * Removes a class and returns a unique list either in array or space separated
+     *
+     * @param array|string $input The array or string to remove the class from
+     * @param array|string $removeClass the new class or classes to remove
+     * @param string $useIndex if you are inputting an array with an element other than default of 'class'.
+     * @return array|string
+     */
+    public function removeClass($input, $removeClass, $useIndex = 'class')
+    {
+        // NOOP
+        if (empty($removeClass)) {
```

Comment:
What if the class is `'0'`?

Reply:
CSS identifiers can't start with a digit, so I don't think it's a problem.

https://www.w3.org/TR/CSS21/syndata.html#characters